sys/net/gnrc/network_layer/sixlowpan/nd/gnrc_sixlowpan_nd.c

```diff
@@ -36,7 +36,7 @@ static inline void _rtr_sol_reschedule(gnrc_ipv6_netif_t *iface, uint32_t sec_de
 
 static inline uint32_t _binary_exp_backoff(uint32_t base_sec, unsigned int exp)
 {
-    return genrand_uint32_range(0, (1 << exp) - 1) * base_sec;
+    return genrand_uint32_range(0, (1 << exp)) * base_sec;
```

Comment:
Also: with this change https://github.com/RIOT-OS/RIOT/pull/3807/files#diff-9b0aedc5fd4543061c862e245fe43c75R72 would need adaption too.